src/main/kotlin/org/koitharu/kotatsu/parsers/site/NicovideoSeigaParser.kt

```diff
@@ -0,0 +1,161 @@
+package org.koitharu.kotatsu.parsers.site
+
+import org.koitharu.kotatsu.parsers.MangaLoaderContext
+import org.koitharu.kotatsu.parsers.MangaParser
+import org.koitharu.kotatsu.parsers.MangaParserAuthProvider
+import org.koitharu.kotatsu.parsers.MangaSourceParser
+import org.koitharu.kotatsu.parsers.config.ConfigKey
+import org.koitharu.kotatsu.parsers.exception.AuthRequiredException
+import org.koitharu.kotatsu.parsers.model.*
+import org.koitharu.kotatsu.parsers.util.*
+import java.util.*
+
+private const val STATUS_ONGOING = "連載"
+private const val STATUS_FINISHED = "完結"
+
+@MangaSourceParser("NICOVIDEOSEIGA", "Nicovideo Seiga", "ja")
+class NicovideoSeigaParser(override val context: MangaLoaderContext) : MangaParser(MangaSource.NICOVIDEOSEIGA),
+	MangaParserAuthProvider {
+
+	override val authUrl: String
+		get() = "https://account.nicovideo.jp/login?site=seiga"
+
+	override val isAuthorized: Boolean
+		get() {
+			return context.cookieJar.getCookies(getDomain()).any {
+				it.name == "user_session"
+			}
+		}
+
+	override suspend fun getUsername(): String {
+		val body = context.httpGet("https://app.nicovideo.jp/my/apps").parseHtml().body()
+		return body.selectFirst("#userinfo > div > div > strong")?.text() ?: parseFailed("Cannot get username")
+	}
+
+	override val sortOrders: Set<SortOrder> = EnumSet.of(
+		SortOrder.UPDATED,
+		SortOrder.POPULARITY,
+	)
+
+	override val configKeyDomain: ConfigKey.Domain = ConfigKey.Domain("seiga.nicovideo.jp", null)
```

Comment:
```suggestion
	override val configKeyDomain: ConfigKey.Domain = ConfigKey.Domain("nicovideo.jp", null)
```

Subdomain can be hardcoded.